Cloud Application Manager/Deploying Anywhere/using-compute-instances.md

```diff
@@ -0,0 +1,68 @@
+"date": "11-05-2019",
+"author": "Sharon Wang",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+**In this article:**
+
+* [Overview](#overview)
+* [Audience](#audience)
+* [Prerequisites](#prerequisites)
+* [Create a Provider](#create-a-provider)
+* [Register a Server](#register-a-server)
+* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
+
+### Overview 
+
+This article is meant to assist users of Cloud Application Manager trying to register their stand-alone/physical machines to CAM. You can bring your own infrastructure to Cloud Application Manager by installing the Cloud Application Manager agent and register it to Compute Instances Provider. This will enable lifecycle of the instance through Cloud Application Manager like with any other natively-deployed instance.
+
+### Audience
+
+All users of Cloud Application Manager who wants to register their stand-alone machines to Compute Instances Provider.
+
+### Prerequisites
+
+* An active **Cloud Application Manager** account
+* Any infrastructure capable of running Cloud Application Manager Agent.
+
+
+### Create a Provider
+
+The preferred method to register any infrastructure in CAM, is to register it into a provider, instead of a stand-alone instance. To learn more about registering your servers as a stand-alone instance, please review the [Deploying Anywhere](./deploying-on-anyinfra.md)
```

Comment:
I will remove this paragraph from here and add it to the `deploying-on-anyinfra.md` article (linking to this one instead)

Reply:
Sorry, you will do it or I should be the one doing it?